### ISSUE TITLE
This needs to be published to NPM.

### ISSUE THREAD

**Issue by marshallswain** [justaprogrammer/ObjectId.js#5]
Great work! This really needs to be on NPM.

I know that it's MIT licensed, but it seems polite to ask, first.  :)

If you don't want to, I can.  I'll just need to add a package.json file and make it 'require'-able.  Or I can submit a pull request with the changes...  Just let me know.

**Comment by zippy1981** [comment]
@marshallswain You are welcome to do it and I will accept a pull request. I just have one question. This is entirely designed to be run from a browser. Are there npms for clientside javascript libraries?

**Comment by marshallswain** [comment]
Yes.  Package managers like webpack and browserify have made them more popular.

Have a great day!

Marshall Thompson

On February 21, 2014 at 8:36:44 AM, Justin Dearing (notifications@github.com) wrote:

@marshallswain You are welcome to do it and I will accept a pull request. I just have one question. This is entirely designed to be run from a browser. Are there npms for clientside javascript libraries?

**Comment by mkrn** [comment]
So what's the status of putting it up on npm?
Really want to try it out 
Thanks!

**Comment by marshallswain** [comment]
I figured out afterward that you can just use npm install [github repo address] to pull directly from your repository.  Try it out.

Have a great day!

Marshall Thompson

**Comment by mkrn** [comment]
Hey @marshallswain I tried it but then realized that the module had straight using "document" in it  so I hacked it around to add browser-checking and then use node's process.pid when it's used on node..

However that didn't work and I've put it off.

I think it would be huge to create a small universal module to use on client and node, because using 
var ObjectID = require("bson/lib/bson/objectid").ObjectID
is polluting browserify build...
